CONTRIBUTING.md

```diff
@@ -16,22 +16,22 @@ It's even better if you know a bit of React and the basics of TypeScript.
 ## Software requirements
 
 - Git
-- Latest version of Node.js 16
-- npm v8 package manager (bundled with Node.js)
+- pnpm package manager (version specified in "engines" property within `package.json`)
+- Latest version of Node.js 16 (optional)
 
 This guide assumes you'll run the code on your workstation directly.
 
 ### Where to get Git
 
 The [git-scm.com](https://git-scm.com/) site has links to download Git for your operating system.
 
-### Where to get Node.js
+### Where to get pnpm
 
-You can install Node.js + npm using a [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm) (recommended).
+In case you didn't get pnpm setup in your environment, [there are different ways of installing/updating it](https://pnpm.io/installation).
 
-### Where to get npm (optional)
+### Where to get Node.js (optional)
 
-In case you didn't get npm v8 automatically bundled with Node.js, [there are different ways of installing/updating it](https://docs.npmjs.com/cli/v7/configuring-npm/install).
+You can install Node.js through a [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm). However, [pnpm can manage the Node.js version automatically for you](https://pnpm.io/blog/2021/12/29/yearly-update#managing-nodejs-versions-since-v6120), so you can run it even with no Node.js preinstalled on the system.
```

Comment:
```suggestion
You can install Node.js through a [Node.js version manager](https://docs.npmjs.com/cli/v7/configuring-npm/install#using-a-node-version-manager-to-install-nodejs-and-npm).
However, [pnpm can manage the Node.js version automatically for you](https://pnpm.io/blog/2021/12/29/yearly-update#managing-nodejs-versions-since-v6120), so you can run it even with no Node.js preinstalled on the system.
```